CHANGELOG.md

```diff
@@ -1,5 +1,165 @@
 # Changelog
 
+## 🎰 1.6.0
+
+- ### Features
+
+  - **_BREAKING CHANGE_: Require `webpack_config` to build with a custom configuration - [EverlastingBugstopper], [issue/296] [pull/847]**
+
+    Wrangler will no longer use a `webpack.config.js` at the root of your project to build your worker. If you would like to continue using a custom build configuration, you will need to specify the `webpack_config` field in your `wrangler.toml`.
```

Comment:
Might be useful to show a TOML syntax example here - for instance, do I do `webpack.config` or `webpack.config.js`? Or `./webpack.config.js`?